test/azure/src/main/java/fixtures/azurespecials/implementation/OdatasImpl.java

```diff
@@ -67,7 +65,6 @@ public OdatasImpl(AutoRestAzureSpecialParametersTestClientImpl client) {
     /**
      * Specify filter parameter with value '$filter=id gt 5 and name eq 'foo'&amp;$orderby=id&amp;$top=10'.
      *
-     * @throws IllegalArgumentException thrown if parameters fail the validation.
```

Comment:
Did you make it so this `@throws` doc is only added if there are actually parameters to validate?

Reply:
I did change it to do that, but later realized that IllegalArgumentException can get thrown by one of the methods that this client method may call. I'm not completely happy with this part and I may look into it further.